### ISSUE TITLE
Scripts with null values that are certain characters are unable to be loaded

### ISSUE THREAD

**Issue by zhangcandrew** [weecology/retriever#736]
Error: 

```
Failed to load script: qq (.retriever/scripts/)
Exception: invalid syntax (qq.py, line 8) 
Failed to load script: qq (.retriever/scripts/)
Exception: invalid syntax (qq.py, line 8) 
```
Where qq is a newly created script using retriever new_json. 
The error is returned due to this line: 

```
tables={'aadsf': Table('aadsf', cleanup=Cleanup(correct_invalid_value, nulls=?),columns=[])}
```
since the null value is a character, it should be nulls = '?' instead of just nulls = ?. Fixing this change however will cause certain tests in travis and appveyor to be violated, so I have created an issue for now.

**Comment by henrykironde** [comment]
@zhangcandrew what branch was that, were you had tried to fix this issue?

**Comment by zhangcandrew** [comment]
I believe it was called newJsonChanges

**Comment by zhangcandrew** [comment]
This issue was resolved after a discussion in #753.
